code/datums/craft/recipes/alchemy.dm

```diff
@@ -83,189 +93,195 @@
+	result = /obj/item/reagent_containers/glass/bottle/potassium
 	steps = list(
 		list(/obj/item/reagent_containers/glass, 1),
-		list(/obj/item/stack/ore, 2), //getting it form ore
+		list(/obj/item/reagent_containers/food/snacks/grown, 3), // Can't make it exclusive to bananas so this is the next best thing.
 		list(QUALITY_HEATING, 5, 10)
 	)
 
-/datum/craft_recipe/alchemy/ammonia
-	name = "Ammonia Bottle"
-	result = /obj/item/reagent_containers/glass/bottle/ammonia
+/datum/craft_recipe/alchemy/radium
+	name = "Radium Bottle"
+	result = /obj/item/reagent_containers/glass/bottle/radium
 	steps = list(
 		list(/obj/item/reagent_containers/glass, 1),
-		list(/obj/item/stack/ore, 2), //getting it form ore
+		list(/obj/item/stack/ore/uranium, 1),
 		list(QUALITY_HEATING, 5, 10)
 	)
 
-/datum/craft_recipe/alchemy/carbon
-	name = "Carbon Bottle"
-	result = /obj/item/reagent_containers/glass/bottle/carbon
+/datum/craft_recipe/alchemy/sacid // Alphabetically by reagent name
+	name = "Sulphuric Acid Bottle"
+	result = /obj/item/reagent_containers/glass/bottle/sacid
 	steps = list(
 		list(/obj/item/reagent_containers/glass, 1),
-		list(/obj/item/reagent_containers/food/snacks/meat/pork, 3),
+		list(/obj/item/stack/ore, 1), // Sulfur from rocks
+		list(/obj/item/reagent_containers/food/snacks/grown, 1), // Hydrogen from biomass
 		list(QUALITY_HEATING, 5, 10)
 	)
 
-/datum/craft_recipe/alchemy/copper
-	name = "Copper Bottle"
-	result = /obj/item/reagent_containers/glass/bottle/copper
+/datum/craft_recipe/alchemy/silicon
+	name = "Silicon Bottle"
+	result = /obj/item/reagent_containers/glass/bottle/silicon
 	steps = list(
 		list(/obj/item/reagent_containers/glass, 1),
-		list(/obj/item/stack/ore, 1), //getting it form ore
+		list(/obj/item/stack/ore/glass, 3),
 		list(QUALITY_HEATING, 5, 10)
 	)
 
-/datum/craft_recipe/alchemy/ethanol
-	name = "Ethanol Bottle"
-	result = /obj/item/reagent_containers/glass/bottle/ethanol
+/datum/craft_recipe/alchemy/sodium
+	name = "Sodium Bottle"
+	result = /obj/item/reagent_containers/glass/bottle/sodium
 	steps = list(
 		list(/obj/item/reagent_containers/glass, 1),
-		list(/obj/item/reagent_containers/food/snacks/grown, 8),
+		list(/obj/item/stack/ore, 3), // Break rock salt into sodium
 		list(QUALITY_HEATING, 5, 10)
 	)
 
-/datum/craft_recipe/alchemy/iron
-	name = "Iron Bottle"
-	result = /obj/item/reagent_containers/glass/bottle/iron
+/datum/craft_recipe/alchemy/sugar
+	name = "Sugar Bottle"
+	result = /obj/item/reagent_containers/glass/bottle/sugar
 	steps = list(
 		list(/obj/item/reagent_containers/glass, 1),
-		list(/obj/item/stack/ore, 1), //getting it form ore
+		list(/obj/item/reagent_containers/food/snacks/grown, 1), // Extracting sucrose from plants
 		list(QUALITY_HEATING, 5, 10)
 	)
 
-/datum/craft_recipe/alchemy/lithium
-	name = "Lithium Bottle"
-	result = /obj/item/reagent_containers/glass/bottle/lithium
+/datum/craft_recipe/alchemy/sulfur
+	name = "Brimstone Bottle" // Old name of Sulfur
```

Comment:
should need an egg

Reply:
Fair point on this one tbh, rotten eggs contain sulfur which gives it their smell